install/depends

```diff
@@ -180,6 +180,7 @@ elif [ "${DISTRO}" = "darwin" ]; then
 		brew "findutils"
 		brew "gawk"
 		brew "gnu-sed"
+		brew "gnu-tar"
```

Comment:
I'm shocked this hadn't been required before...

Reply:
So `install/depends` is meant to be a minimal set of packages required for core ebash functionality. There is `bin/recommends` which is meant to be additional packages to be installed on top of the base ones. ebash's `archive` module definitely requires `gnu-tar`, but it's something we didn't use on Mac so didn't make it a core requirement. 

That said, I like this change. Makes Mac more consistent with other distros.

But I think we should remove remove `gnu-tar` from `install/recommends` since it is now covered by `install/depends`

Reply:
And yeah, tar is provided by all Linux. You might want to check out `tests/all_depends.etest` which makes sure all dependencies are installed. There is a `ETEST_install_go` you. might consider adding another unit test to that one to exercise your new code (not a mock test but one which will get run on all distros in matrix build)

Reply:
Oh that's a really good idea, okay I'll do that, thanks!